### PR TITLE
3DFileViewer+Base: Add tuba

### PR DESCRIPTION
**Pull request by sunverwerth**
grafik

Adds a free range, organic, handmade tuba to the 3D Models folder. Also fixes a small bug in 3DFileViewer.

### CODE REVIEW

**Comment by GMTA** [comment]
This is tootally tubalicious. LGTM!

**Comment by sunverwerth** [comment]
We already have pipes. So I think this fits right in.

**Comment by sunverwerth** [comment]
This is tootaly a tuba, not some inferior instrument like a trumpet or sth

**Comment by Quaker762** [comment]
This is a good start, however we seem to be missing a top-hat, as well as a baseball cap.

@awesomekling could you advise on the colours need here??

Tag @GMTA @sunverwerth

File: Userland/Applications/3DFileViewer/main.cpp [hunk 1]
```diff
@@ -299,13 +299,12 @@ bool GLContextWidget::load_file(Core::File& file)
             texture_image = bitmap_or_error.release_value_but_fixme_should_propagate_errors();
     } else {
         auto response = FileSystemAccessClient::Client::the().try_request_file(window(), builder.string_view(), Core::OpenMode::ReadOnly);
-        if (response.is_error())
```
Comment (adamrt) [inline Userland/Applications/3DFileViewer/main.cpp:302]: Sweet Tuba! Any reason you nested this code instead of leaving it nice and flat?

Comment (sunverwerth) [reply Userland/Applications/3DFileViewer/main.cpp:302]: This is to make sure we still load the 3d model if the texture could not be loaded. Notice: I removed the early return.

**Comment by sunverwerth** [comment]
> This is a good start, however we seem to be missing a top-hat, as well as a baseball cap.
> 
> @awesomekling could you advise on the colours needed for this??
> 
> Tag @GMTA @sunverwerth

:yakthonk:

File: Userland/Applications/3DFileViewer/main.cpp (hunk 1 above)
Comment (adamrt) [reply Userland/Applications/3DFileViewer/main.cpp:302]: Sorry, I hadn't had coffee and misread the indentation level. Nice work!

**Comment by awesomekling** [comment]
Uhhh

**Pull request merged**